wp_api_integration_tests/tests/test_update_checks_immut.rs

```diff
@@ -0,0 +1,29 @@
+use std::sync::Arc;
+
+use serial_test::parallel;
+use wp_api::wordpress_org::client::WordPressOrgApiClient;
+use wp_api_integration_tests::{api_client, test_site_url, AsyncWpNetworking, TestCredentials};
+
+#[tokio::test]
+#[parallel]
+async fn plugins_update_check() {
+    let rest_client = api_client();
+    let plugins = rest_client
+        .plugins()
+        .list_with_view_context(&Default::default())
+        .await;
+    let plugins = plugins.unwrap().data;
```

Comment:
Nitpick for consistency:

```suggestion
use wp_api_integration_tests::{
    api_client, test_site_url, AssertResponse, AsyncWpNetworking, TestCredentials,
};

#[tokio::test]
#[parallel]
async fn plugins_update_check() {
    let plugins = api_client()
        .plugins()
        .list_with_view_context(&Default::default())
        .await
        .assert_response()
        .data;
```